### ISSUE TITLE
Network not stable 

### ISSUE THREAD

**Issue by cho88hk** [akridge/Oracle-Cloud-Free-Valheim-Game-Server-Setup#2]
Disconnect within few min. , any idea? Thanks.
[Info   : Unity Log] 03/28/2021 22:17:55: ZNet Shutdown

[Info   : Unity Log] 03/28/2021 22:17:55: Sending disconnect msg

[Info   : Unity Log] 03/28/2021 22:17:55: Released session ticket

[Warning: Unity Log] 03/28/2021 22:17:56: Local player destroyed

[Info   : Unity Log] 03/28/2021 22:17:56: Lost connection to server:ErrorDisconnected

[Info   : Unity Log] 03/28/2021 22:17:56: Lost connection to server:ErrorDisconnected

[Info   : Unity Log] 03/28/2021 22:17:57: ZNet OnDestroy

[Info   : Unity Log] 03/28/2021 22:17:57: Net scene destroyed

**Comment by akridge** [comment]
Seems to be a problem with a recent Valehim patch. They updated the netcode 
 in order to help reduce desync issues, but now its introduced disconnect issues for many users. 

Server restart is recommend to help. Also making sure steam and valheim are both updated. 

https://valheim-server-help.github.io/serverTroubleshooting/

**Comment by raspu** [comment]
Similar issue here, both clients and server are up to date.

**Comment by akridge** [comment]
Yeah this is just the nature of the game being early access at the moment. They tried to help desync and added disconnect issues. Hopefully a patch will come soon to fix the disconnects to clients with spotty connections. 

If you're looking for a work around in the mean time then check out Valheim+. The V+ mod does offer some configurations to alleviate this, but nothing so far is 100%.
